### DISCUSSION TITLE
0.1uF Cap needed on final PCB?

### DISCUSSION THREAD

**Discussion by RossAWaddell** [SpenceKonde/ATTinyCore#877] [Support / Q & A]
The Ref_wiring.md file says you need a 0.1uF cap between Vcc and GND as close to the chip/pins as possible. Is that also needed on the final PCB, or just during programming (I'm using an ATtiny85)?

Also, I'm planning on using SparkFun's Tiny AVR Programmer ([https://learn.sparkfun.com/tutorials/tiny-avr-programmer-hookup-guide/all#programming-in-arduino](url) to program my ATtiny85 - will this be a problem with this core? I'm using the official Arduino.cc app (v1.8.18) and this in the Additional Boards Manager URL: https://raw.githubusercontent.com/damellis/attiny/ide-1.6.x-boards-manager/package_damellis_attiny_index.json. I've also added the ATtiny boards, although the only one I found in Boards Manager was from David Mellis ([https://github.com/damellis/attiny](url))

**Comment by hmeijdam** [comment]
Using the 0.1Uf cap is always recommended. Not just for programming.

That programmer is a UsbtinyISP programmer so should work with Attinycore.

The damellis core is a different and much older core than AttinyCore.  Did you read the[ installation instructions](https://github.com/SpenceKonde/ATTinyCore/blob/v2.0.0-devThis-is-the-head-submit-PRs-against-this/Installation.md#boards-manager-installation) for this core?

**Reply by RossAWaddell** [reply]
I did but assumed it was the same as this. I guess I need to change the URL in Additional Boards Manager to what you have here?

EDIT: I replaced the URL with this one and now I see the correct board file. All good.

**Reply by RossAWaddell** [reply]
… and I’ve added a 0.1uF ceramic cap to my board design.
